src/Arcus.EventGrid.Tests.Unit/Parsing/EventParsingTests.cs

```diff
@@ -18,11 +18,11 @@ public void ParseAsCloudEvent_ValidBlobCreatedEvent_ShouldFail()
             string rawEvent = EventSamples.BlobCreateEvent;
 
             // Act
-            var eventGridEventBatch = EventGridParser.Parse(rawEvent);
+            var EventBatch = EventGridParser.Parse(rawEvent);
```

Comment:
Could you use a camel case here for the name? I guess you wanted to use a typed variable declaration here?

Reply:
ah slipped through, fixed now.